couchpotato/core/plugins/userscript/template.js_tmpl

```diff
@@ -100,7 +100,7 @@ var osd = function(){
 
         // Try and get imdb url
         try {
-            var regex = new RegExp(/tt(\d{7})/);
+            var regex = new RegExp(/tt(\d{8})/);
```

Comment:
I'm not sure about using a fixed 8 size value.
Why not just validate digits or using same pattern as python ?